documentation/src/main/markdown/releasenotes.md

```diff
@@ -12,6 +12,7 @@
 * Addition of command line help option, -hyaml, to generate a template configuration file.
 * [solution_name]'s generated air gap zip is uploaded to Artifactory under the name "synopsys-detect-<version>-air-gap-no-docker.zip". Older naming patterns for this file are no longer supported.
 * Failures in detectors will now be reported in the console output using the ERROR logging level. The ERROR logging is also used if there are errors in the overall status.
+* Evicted dependencies in Simple Build Tool(SBT) projects will no longer be included in the Bill of Materials(BoM) generated during the scan.
```

Comment:
This should also go under a new 8.8.0 section and be indicated as a resolved issue in addition to the note you have under Change features. Thanks.
Something like:

## Version 8.8.0

### Resolved issues

* (IDETECT-3111) When scanning SBT projects, "Evicted" dependencies are appearing in the resulting BOM when they should be excluded.

### Changed features

* Evicted dependencies in Simple Build Tool(SBT) projects will no longer be included in the Bill of Materials(BoM) generated during the scan.